src/main/java/com/google/cloud/tools/maven/cloudsdk/CloudSdkMojo.java

```diff
@@ -42,6 +42,9 @@
   @Parameter(property = "serviceAccountKeyFile", required = false)
   private File serviceAccountKeyFile;
 
+  /** Optional parameter to configure the gcloud logging verbosity level. */
+  @Parameter private String verbosity;
```

Comment:
Is it unnecessary to have
```
@Parameter(property = "verbosity", required = false)
```
like for the `serviceAccountKeyFile` field?

Reply:
property is for the command line option `-Dverbosity=info` or something, which I'm hesitant to add, maybe it could be `gcloud.verbosity` or something? `verbosity` seems like it could clash with a bunch of things

Reply:
I will add `gcloud.verbosity`